plugins/services/src/js/components/ServiceConfigBaseSectionDisplay.js

```diff
@@ -44,7 +63,7 @@ class ServiceConfigBaseSectionDisplay extends React.Component {
 
       if (row.render != null) {
         // If a custom render method was specified on the row, we use that.
-        return row.render(value, appConfig);
+        return row.render(value, appConfig, editLink);
```

Comment:
I have a bad feeling about passing down the entire element. I would prefer if we passed down the section name and the callback function, letting the leaf element to decide how to render everything.